### PR TITLE
Update extension link

### PR DESCRIPTION
**Pull request by erin-at-work**
Fixes 
https://github.com/coinbase/coinbase-wallet-sdk/issues/407#issue-1168933381

### CODE REVIEW

File: src/components/TryExtensionLinkDialog.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ export const TryExtensionLinkDialog: FunctionComponent<{
           <TryExtensionBox
             onInstallClick={() => {
               window.open(
-                "https://api.wallet.coinbase.com/rpc/v2/desktop/chrome",
```
Comment (jeongmincho) [inline src/components/TryExtensionLinkDialog.tsx:76]: oof link rot?

File: src/components/TryExtensionLinkDialog.tsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export const TryExtensionLinkDialog: FunctionComponent<{
...
+                "https://chrome.google.com/webstore/detail/coinbase-wallet-extension/hnfanknocfeofbddgcijnmhnfnkdnaad",
```
Comment (jeongmincho) [inline src/components/TryExtensionLinkDialog.tsx:76]: nice: https://support.google.com/chrome/a/answer/6177447?hl=en

**Review by jeongmincho — APPROVED** [review]

File: src/components/TryExtensionLinkDialog.tsx (hunk 2 above)
Comment (jeongmincho) [reply src/components/TryExtensionLinkDialog.tsx:76]: Screen Shot 2022-03-14 at 5 55 40 PM

**Comment by erin-at-work** [comment]
this is an api issue

**Pull request closed without merging**